docs/notebooks/entropic_map_estimator.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>Please use <code>&lt;cite data-cite="pooladian:21"&gt;&lt;/cite&gt;"</code> (also in other places) when referring to the paper, as this uses the references.bib.</p><p>I'd slightly modify the last sentence to <code>... taking advantage of the EntropicPotentials</code>class.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>